ts/components/conversation/Message.tsx

```diff
@@ -307,11 +307,12 @@ export class Message extends React.PureComponent<Props, State> {
     return (
       <div className="module-message__author">
         <ContactName
-          phoneNumber={authorPhoneNumber}
+          phoneNumber={`(...${authorPhoneNumber.substring(authorPhoneNumber.length - 6, authorPhoneNumber.length)})`}
```

Comment:
I believe `authorPhoneNumber.substring(authorPhoneNumber.length - 6)` gives the same result.